src/main/java/org/folio/services/item/ItemService.java

```diff
@@ -176,4 +185,27 @@ public ItemService(Context vertxContext, Map<String, String> okapiHeaders) {
   private Function<SQLConnection, Future<RowSet<Row>>> updateSingleItemBatchFactory(Item item) {
     return connection -> itemRepository.update(connection, item.getId(), item);
   }
+
+  private Future<Item> refuseWhenStatisticalCodeIdDoesNotExist(final Item item) {
```

Comment:
Can you add documentation what happens when a code is deleted from statistical_code table that is still in use in statisticalCodeIds field of an instance, holding or item, or open a Jira for that task?

Reply:
Preferably, it would not allow for it to be deleted if in use and constrained so at the database level. Since that is unlikely the case, I assume some UI feature will break.

Reply:
Moved to issue.